### ISSUE TITLE
Looks like the Jubilee carraige might not fit our bridge. 

### ISSUE THREAD

**Issue by mikeabuilder** [BainbridgeArtisanResourceNetwork/Mark4_printer#22]
One design goal for the Mark 4 is to be able to use the Jubilee carriage if we want. There are a few things that look like they might not be compatible. Need to verify the latest Jubilee design will work. 

1. Position of the belts (x, y, and z directions) relative to the fastener on the Jubilee carriage.  May need to change the pulley positions on the bridge, and/or the pulley standoff heights.
2. Alignment of the mounting holes to the X-Rail block. Jubilee uses a "C" style (Compact) block.
3. X-endstop "target". The latest Jubilee design uses an extension of the standoff for the "high pulley" on the front of the left side of the bridge as a place for the X-endstop microswitch to hit. Our current design has the "low pulley" at the front so there is no standoff to extend into the endstop target. May need to swap high and low pulleys in our design.

See picture. Jubilee in back, Mark design (in progress) in front. 
![Jubilee carriage vs Mark 4](https://user-images.githubusercontent.com/57372027/89111970-7fb01480-d411-11ea-9439-13e4d896c2df.jpg)

**Comment by mikeabuilder** [comment]
I've done a close comparison of the Jubilee carriage design already in our Fusion model (the "OLD" jubilee carriage) and the latest that was posted to Jubilee's github.  I've attached a pdf of the comparison of the major parts, but the conclusion is that although there are some changes in the new Jubilee carriage design, the major interfaces (Carriage to X-axis sliding block, Carriage mounting plate to tool, and carriage back plate to belts) nave not changed. This meanss no changes to pulley locations are needed.  I have swapped the "high" and "low" pulley positions in our model. The A belt is not the higher belt and the B belt is lower. This will allow us to build an x-Axis microswitch target into one of the belt standoffs, like the Jubilee. 
[OLDER ve NEWER Jubilee Carriage.pdf](https://github.com/BainbridgeArtisanResourceNetwork/Mark4_printer/files/5031563/OLDER.ve.NEWER.Jubilee.Carriage.pdf)
